### PR TITLE
Include operation for clear raw data buffer

### PR DESCRIPTION
**Pull request by joaobrittoneto**
Deals with https://github.com/Brazilian-Institute-of-Robotics/drivers-orogen-usbl_evologics/issues/26 in syskit layer, using this new operation

### CODE REVIEW

**Comment by joaobrittoneto** [comment]
@g-arjones, any comment here?

File: tasks/Task.hpp [hunk 1]
```diff
@@ -75,6 +75,11 @@ namespace usbl_evologics {
          */
         virtual void clearTransmissionBuffer(void);
 
+        /** Drop burst data and terminate the acoustic connection
+         *
+         */
+        virtual void clearRawDataBuffer(void);
```
Comment (g-arjones) [inline tasks/Task.hpp:81]: Why is this virtual?

Comment (joaobrittoneto) [reply tasks/Task.hpp:81]: It's the implementation of the operation defined in the .orogen file. It automatically generated a virtual method in the BaseTask, that is overridden in Task.

Comment (g-arjones) [reply tasks/Task.hpp:81]: Yes, of course... Looks ok to me.

Comment (joaobrittoneto) [reply tasks/Task.hpp:81]: :+1:

**Comment by joaobrittoneto** [comment]
Merging

**Pull request merged**